cmd/list/region/cmd.go

```
@@ -80,6 +89,16 @@ func run(cmd *cobra.Command, _ []string) {
 		os.Exit(1)
 	}
 
+	hypershiftEnabled, err := r.OCMClient.IsCapabilityEnabled(ocm.HypershiftCapability)
+	if err != nil {
+		r.Reporter.Errorf("%v", err)
+		os.Exit(1)
+	}
+
+	if hypershiftEnabled {
+		regionsWithHostedCPSupport, _ = r.OCMClient.ListHostedCPSupportedRegion()
```

why do we need to ignore the error

---

fixed, thanks